test/key_test.py

```diff
@@ -25,25 +24,29 @@ def setUp(cls):
             pygame.display.init()
 
     def test_import(self):
-        "does it import"
+        """does it import?"""
         import pygame.key
 
-    @unittest.skipIf(SDL1, "SDL1 always thinks it has keyboard focus.")
+    # fixme: test_get_focused always fails in linux
+    @unittest.skipIf(
+        sys.platform.startswith("linux"),
+        "This test fails in all tested GNU/Linux"
+    )
     def test_get_focused(self):
+        # Note: this test fails in SDL2 and it was skipped in SDL1.
+        # There might be something wrong in this function
```

Comment:
I don't think it's always broken on Linux, and it works fine on two other platforms. I think this is a problem with your system. 

I'm not sure this test should be skipped.

Reply:
I removed the "skip". However, I left the `# fixme` because the reason why it fails it is still unknown.